python/ray/tests/test_output.py

```diff
@@ -181,6 +183,9 @@ def f():
     indirect=True,
 )
 def test_autoscaler_infeasible(ray_start_cluster_head_with_env_vars):
+    if os.environ.get("RAY_enable_autoscaler_v2") == "1":
```

Comment:
Do we still need to use `pytest.skip` if we have already added the `@skip_flaky_core_test_premerge decorator`?

Reply:
oops, sorry this was errant!